src/proximity-manager/index.ts

```diff
@@ -325,6 +326,30 @@ const ProximityManager = () => {
     }
   };
 
+  // TODO: this function is used to test COSE sign
+  // on a mocked plaintext. It should be removed
+  // or moved where appropriate.
+  const signMessage = async (message: string) => {
+    try {
+      const headers = {
+        p: { alg: 'ES256' },
+        u: { kid: '11' },
+      };
+      const signer = {
+        key: {
+          d: Buffer.from(
+            '6c1382765aec5358f117733d281c1c7bdc39884d04a45a1e6c67c858bc206c19',
+            'hex'
+          ),
+        },
+      };
+      const buf = await sign.create(headers, message, signer);
+      console.log('Signed message: ' + buf.toString('hex'));
+    } catch (error) {
+      console.log(error);
+    }
+  };
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏What does this function does? Why it returns nothing?